Assets/Tests/InputSystem/Plugins/XRTests.cs

```diff
@@ -728,6 +728,37 @@ public void Components_TrackedPoseDriver_RetainsPoseWhenTrackedDeviceRemoved()
         }
     }
 
+    [Category("Components")]
+    public void Components_TrackedPoseDriver_RetainsPoseWhenNoActionIsBound()
+    {
+        // Tests/reproduces the scenario described in https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-699
+        // i.e. that rotation and/or position is not updated if device is not connected and track state isn't ignored.
```

Comment:
```suggestion
    [TestCase(false)]
    [TestCase(true)]
    public void Components_TrackedPoseDriver_RetainsPoseWhenNoActionIsBound(bool ignoreTrackingState)
    {
        // Tests/reproduces the scenario described in https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-699
        // i.e. that rotation and/or position is not updated if device is not connected.
```
This is actually the case whether `ignoreTrackingState` is `false` or `true`. This suggestion adds two `TestCase`. Combine this suggestion with the other one that uses the test parameter.